src/UI/examples/Layout/Page/Standard/ui.php

```diff
@@ -23,7 +23,9 @@ function ui()
 	$content = pagedemoContent($f);
 	$metabar = pagedemoMetabar($f);
 	$mainbar = pagedemoMainbar($f, $renderer)
-		->withActive("pws");
+		->withActive("pws")
+		//->withActive("tool2")
```

Comment:
Please put a comment around this uncommented code that tells why it is there. This just looks as if this was forgotten here...